gm4_live_catch/data/gm4/advancements/live_catch.json

```diff
@@ -0,0 +1,29 @@
+{
+  "display": {
+    "icon": {
+      "item": "pufferfish",
+      "nbt": "{CustomModelData:1}"
+    },
+    "title": {
+      "translate": "%1$s%3427655$s",
+      "with": [
+        "Watch Out!",
+        {"translate": "advancement.gm4.live_catch.title"}
+      ]
+    },
+    "description": {
+      "translate": "%1$s%3427655$s",
+      "with": [
+        "Catch a pufferfish and have it fly into your face.",
```

Comment:
This description is too wordy in my opinion. It should say something like "Catch a pufferfish to the face"